### PR TITLE
Added react-native plugin

### PR DESCRIPTION
**Pull request by maximilianhurl**

### CODE REVIEW

**Comment by RealOrangeOne** [comment]
Probably not ideal seeing as most of our projects aren't react-native based?

**Comment by JakeSidSmith** [comment]
You can extend eslint config in your project. :D

```
{
    "extends": "./node_modules/eslint-config/.eslintrc",

    "rules": {
        "plugins": [
          "react-native"
        ]
    }
}
```

**Comment by maximilianhurl** [comment]
Okay well maybe I'll leave it

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
   "homepage": "https://github.com/dabapps/eslint-config",
   "dependencies": {
     "eslint": "=1.5.0",
-    "eslint-plugin-react": "=3.4.2"
+    "eslint-plugin-react": "=3.4.2",
+    "eslint-plugin-react": "=0.5.0"
```
Comment (weargoggles) [inline package.json]: this looks like a duplicate dependency

**Pull request closed without merging**